back-end/app/routes.py

```diff
@@ -199,7 +202,101 @@ def update_profile():
 
     return jsonify({'message': 'Profile updated successfully'}), 200
 
+@auth_bp.route('/add-user', methods=['POST'])
+@jwt_required()
+def add_user():
+    current_user = get_jwt_identity()  # Get the current logged-in user's identity (username or email)
+    user_name = request.form.get('user_name')
+    role = request.form.get('role')
+    image = request.files.get('image')
```

Comment:
Please validate the file type to ensure only allowed image formats (e.g., PNG, JPG) are uploaded.

Reply:
it's better now